firefox-ios/Client/Frontend/Browser/MainMenu/Redux/MainMenuDetailState.swift

```diff
@@ -14,6 +14,7 @@ struct MainMenuDetailsState: ScreenState, Equatable {
     var shouldGoBackToMainMenu: Bool
     var navigationDestination: MenuNavigationDestination?
     var submenuType: MainMenuDetailsViewType?
+    var currentTabInfo: MainMenuTabInfo?
```

Comment:
I'm not sure I understand why we're adding this here? I don't see it being used anywhere, and, also, if we need it in the reducer, we can always grab it from the `MainMenuState`.